src/scss/components/button/_index.scss

```diff
@@ -1 +1,315 @@
+  }
+
+  @mixin secondary() {
+    border: 1px solid $cyan-900;
+    color: $cyan-900;
+
+    &:hover {
+      background-color: $cyan-500;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:active {
+      background-color: $cyan-700;
+      border-color: transparent;
+      color: $gray-0;
+    }
+
+    &:focus {
+      box-shadow: $focus-glow;
+    }
+
+    &[disabled] {
+      background-color: transparent;
+      border-color: $gray-400;
+      color: $gray-400;
+    }
+  }
+
+  @mixin tertiary() {
+    background-color: transparent;
+    border: none;
+    color: $cyan-900;
+    padding-left: 0;
+    padding-right: 0;
+
+    &:hover {
+      background-color: transparent;
+      border-color: transparent;
+      color: $cyan-500;
+    }
+
+    &:active {
+      background-color: transparent;
+      border-color: transparent;
+      color: $cyan-700;
+    }
+
+    &:focus {
+      box-shadow: $focus-glow;
+    }
+
+    &[disabled] {
+      background-color: transparent;
+      border-color: transparent;
+      color: $gray-400;
+    }
+  }
+
+  @mixin small() {
+    padding: 0.125rem 0.5rem; // 2px 8px
+  }
+
+  @mixin medium() {
+    font-size: 0.875rem;
+    padding: 8px 12px;
+  }
+
+  @mixin large() {
+    font-size: 1rem;
+    padding: 1rem;
+  }
+}
+
+/// ==================================================
+///  COMPONENT STYLES
+/// ==================================================
+
+.hxBtn,
+a[href].hxBtn {
+  @include button-reset();
+  @include button-common();
+  @include button-secondary();
+  @include button-medium();
+
+  // ----- SIZE VARIANTS -----
+  &.hxSm {
+    @include button-small();
+  }
+
+  &.hxLg {
+    @include button-large();
+  }
+
+  // ----- STYLE VARIANTS -----
+  &.hxPrimary {
+    @include button-primary();
+  }
+
+  &.hxTertiary {
+    @include button-tertiary();
+  }
+}//.hxBtn
+
+// ========== BUTTON GROUPS ==========
+.hxBtnGroup {
+  border-radius: 2px;
+  display: inline-flex;
+
+  // Allow focus glow to appear
+  &:not(.hxTertiary) {
+    overflow: hidden;
+  }
+
+  > .hxBtn {
+    border-radius: 0;
+
+    + .hxBtn {
+      border-left: 0;
+    }
+  }
+
+  &.hxPrimary {
+    > .hxBtn {
+      @extend .hxBtn.hxPrimary;
+
+      + .hxBtn {
+        border-left: 1px solid $gray-0;
+      }
+    }
+  }
+
+  &.hxTertiary {
+    > .hxBtn {
+      @extend .hxBtn.hxTertiary;
+
+      + .hxBtn {
+        margin-left: 1.25rem;
+      }
+    }
+  }
+
+  &.hxLg {
+    > .hxBtn {
+      @extend .hxBtn.hxLg;
+    }
+  }
+
+  &.hxSm {
+    > .hxBtn {
+      @extend .hxBtn.hxSm;
+    }
+  }
+}//.hxBtnGroup
+
+// ========== COG BUTTON ==========
+.hxBtn {
+  &.hxCog {
+    border: none;
+    padding: 0.25rem;
+
+    hx-icon {
+      color: $gray-900;
+      height: 1rem;
+      width: 1rem;
+    }
+
+    &:hover, &:active  {
+      background-color: transparent;
+
+      hx-icon {
+        color: $gray-700;
+      }
+    }
+
+    &:focus {
+      box-shadow: 0 0 4px 0 fade($cyan-500, 50%);
```

Comment:
`fade()` needs to be replaced with the SCSS function `rgba()`

```scss
box-shadow: 0 0 4px 0 rgba($cyan-500, 0.5);
```

Reply:
Changes have been done, please review.